main.html

```diff
@@ -5,50 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="main.css" rel="stylesheet">
-    <title>Presentation</title>
-    <style>
-        @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
-        </style>
+    <title>Document</title>
 </head>
 <body>
-    <div class="top-navigation">
-        <div class="item-navigation">
-            <div class="frame-4">
-                <p>Portfolio</p>
-            </div>
-            <div class="frame-3">
-                <p>About</p>
-            </div>
-            <div class="frame-5">
-                <p>Contact</p>
-            </div>
+    <div class="container">
+    <header class="my-header">
+        <div class="time">
+            <img src="mobile_menu_images/Bars/Status-Bars/Time-Style.jpg" alt="Header image">
         </div>
-        <div class="logo-placeholder">
-            <p>My Logo</p>
+        <div class="status-bar">
+            <img src="mobile_menu_images/Bars/Status-Bars/Group3.jpg" alt="Bar Status icons">
         </div>
-    </div>
+    </header>
     
-        <div class="social-media-icons">
-            <img src="images/social-media-icons.jpg">
+    <div class="rectangle"><img src="mobile_menu_images/Bars/Status-Bars/Rectangle.jpg" alt="Divider"></div>
+
+    <nav class="main-menu">
+      <div class="menu-container">
+        <div class="logo-frame">
+            <p>SIKO NORBERT</p>
         </div>
-        <div class="contact-form-background-shapes">
-            <img src="images/contact-form-background-shapes@2x.jpg">
+        <img src="mobile_menu_images/Normal-Button/Tertiary/Enabled.jpg" alt="Button">
+      </div>
+    </nav>
+
```

Comment:
- [ ] It would be better if the container of the toolbar is a `header` tag instead of a `nav` tag. However, you should wrap your navigation links inside a `nav` tag.
- [ ] You logo should have a <a> link tag, apply this chang for design conventions and semantics tags.
**Hint**:
```
<header>
   <a>Logo name</a>
   <nav>
       Your navigation links
   </nav>
</header>
```
- [ ] OPTIONAL: for a better design, use one of your names as a logo name to make it match more the design.